pygqlc/GraphQLClient.py

```diff
@@ -358,7 +379,7 @@ def batchQuery(self, label='query'):
 
   # * END BATCH function **************************************
   # * helper methods
-  def addEnvironment(self, name, url=None, wss=None, headers={}, default=False):
+  def addEnvironment(self, name, url=None, wss=None, headers={}, default=False, timeoutWebsocket=None):
```

Comment:
Mismo comentario que antes (añadir un timeout estándar)
```suggestion
  def addEnvironment(self, name, url=None, wss=None, headers={}, default=False, timeoutWebsocket=60):
```

Reply:
👍